src/main/java/sirius/biz/tenants/packages/Packages.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tenants.packages;
+
+import com.typesafe.config.Config;
+import sirius.biz.model.PermissionData;
+import sirius.kernel.Sirius;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.nls.NLS;
+import sirius.kernel.settings.Extension;
+import sirius.web.security.UserContext;
+
+import java.util.List;
+
+/**
+ * This helper class is a wrapper for the <tt>security.packages</tt> configuratuions.
```

Comment:
please add some lines of documentation of how the wohle sysgtem works and what eacht entity is.

Also add a README.md for the (java) package providing some insight.